scripts/translate_schemas.py

```diff
@@ -33,14 +33,15 @@ def translate_schemas(runner_directory):
             output_dir = f"{runner_directory}/schemas/{language}"
             language_dir = f"{runner_directory}/translations/{language}"
 
-            logger.info("Building %s/%s", relative_dir, schema_file)
+            logger.info("\n-------"
```

Comment:
A space is needed at the end here, otherwise the output looks like:
```
-------Building schemas/eo/census_individual_gb_nir.json
```

Reply:
Odd mine interpreted it as a new line so:`
```
-------
Building schemas/eo/census_individual_gb_nir.json
....
```

Reply:
or maybe not 😂, fixed.